snirf_specification.md

```diff
@@ -135,61 +135,61 @@ HDF5 location paths to denote the indices of sub-elements when multiplicity pres
 
 |  SNIRF-formatted NIRS data structure  |            Meaning of the data               |     Type       |
 |---------------------------------------|----------------------------------------------|----------------|
-| `/formatVersion`                      | * SNIRF format version                       |   `"s"`      * |
```

Comment:
usually I would avoid any white-space changes in any of the commits, by comparing the below edits, I don't see these changes were necessary - can you explain what was modified? newline?

Reply:
I figured out how this happened. I edited the file using the GitHub web editor and it made the changes automatically. They must have changed their web app recently. I'll manually revert

Reply:
Fixed. That was terribly embarrassing.